packages/modeling/src/geometries/geom2/clone.test.js

```diff
@@ -4,7 +4,7 @@ import { clone, create, fromPoints } from './index.js'
 
 test('clone: Creates a clone on an empty geom2', (t) => {
   const expected = {
-    sides: [],
+    outlines: [],
```

Comment:
Didn't clone() change?

Reply:
The test changed so I thought that the clone() function must have changed as well.

Reply:
I assumed that clone() changed because the test has changed.

Reply:
Clone didn't need to change because it just copies the whole object structure, regardless of the internal representation.
```js
export const clone = (geometry) => Object.assign({}, geometry)
```